### ISSUE TITLE
Training time for new data

### ISSUE THREAD

**Issue by agunapal** [omni-us/squeezedet-keras#10]
Hi, 
I am wondering how long it took you to train the model . How many epochs did you have to run it for. I'm trying to run it on the coco dataset. After 150 epochs, I see the loss is still reducing but the eval.py script shows the model getting worse. I see the model generating too ,many bounding boxes. Any help would be appreciated.
Thanks.

**Comment by AjJordy** [comment]
Can you share your git ? I'm trying to train on coco too, but I'm having problem with reshape of the output, it's returning error.

**Comment by AjJordy** [comment]
Hi,
Can you share your project, or give me some help ? I'm trying to train on coco too, but I'm having problems with the reshape the output. It's returning an error.
